### PR TITLE
Catch invalid resolution in CONFIG.ROT

### PR DESCRIPTION
**Pull request by erysdren**

### CODE REVIEW

File: rott/rt_cfg.c [hunk 1]
```diff
@@ -428,6 +428,14 @@ boolean ParseConfigFile (void)
       ReadInt("ScreenWidth", &iGLOBAL_SCREENWIDTH);
       ReadInt("ScreenHeight", &iGLOBAL_SCREENHEIGHT);
 
+	if ((iGLOBAL_SCREENWIDTH != 320 && iGLOBAL_SCREENWIDTH != 640) ||
```
Comment (fabiangreffrath) [inline rott/rt_cfg.c:431]: This would still let 640x200 pass through?

Comment (erysdren) [reply rott/rt_cfg.c:431]: Oh, derp

**Comment by erysdren** [comment]
hang on, i'm working too fast here

**Comment by erysdren** [comment]
now this seems to work as expected. it passes 320x200 and 640x480, but disallows 320x480 and 640x200 and anything else.

**Review by fabiangreffrath — APPROVED** [review]
Thanks!

**Pull request merged**